__tests__/buildJs/pathToKebabCase.test.js

```diff
@@ -1,4 +1,4 @@
-const pathToKebabCase = require('../build').pathToKebabCase
+const pathToKebabCase = require('../../build').pathToKebabCase
```

Comment:
While we're doing this.. can we switch to restructuring rather than assignment through notation?

```suggestion
const {pathToKebabCase} = require('../../build')
```